i18n/sync_tx_src.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+const transifex = require('transifex');
+
+// Globals
+const PROJECT = 'scratch-editor';
+const RESOURCE = 'blocks';
+const TX = new transifex({
+  project_slug: PROJECT,
+  credential: 'api:' + process.env.TX_TOKEN
+});
+
+let en = fs.readFileSync(path.resolve(__dirname, '../msg/json/en.json'));
+en = JSON.parse(en);
+
+if (!process.env.TX_TOKEN) {
+  console.log('ERROR: You must be a Scratch Team member and set your TX_TOKEN to sync with Transifex');
```

Comment:
I think `.log` goes to `stdout`, but I think we should probably correctly write to `stderr` with `console.error` to ensure the messages are output correctly when we have this running in an automated system.
```suggestion
  console.error(new Error('You must be a Scratch Team member and set your TX_TOKEN to sync with Transifex'));
```